conans/client/generators/cmake_multi.py

```diff
@@ -14,6 +14,19 @@ def extend(cpp_info, config):
     if config_info:
         def add_lists(seq1, seq2):
             return seq1 + [s for s in seq2 if s not in seq1]
+
+        def add_dicts(dict1, dict2):
```

Comment:
Maybe this is a bit more idiomatic and efficient:

```python
def add_dicts(d1, d2):
    result = d1.copy()
    for k, v in d2.items():
        existing = result.get(k)
        if existing is None:
            result[k] = v
        else:
            existing.extend((item for item in v if item not in existing))
    return result
```

Reply:
ok, I have to check the order here as it is especially important to keep the behavior